.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: java
 jdk:
-- openjdk6
-- oraclejdk7
+- openjdk7
```

Comment:
not related to pr, but new travis default image (ubuntu trusty) does not have openjdk6 and oraclejdk7 was replaced by openjdk7.   Since we have jenkins and appveyor builds running for java 6 i think it's ok to rely on them for that.